backupper-core/src/main/java/pl/gda/pg/eti/autyzm/backupper/core/FileBackupper.java

```diff
@@ -24,4 +24,50 @@
     @Override public Optional<Backup> getBackup(String backupName) throws BackupperException {
         return null;
     }
+    public static void createFolder(String backUpName) throws IOException {
+        File dataFolder = new File("data");
+        if (!dataFolder.exists()) {
+            dataFolder.mkdir();
+        }
+        File fullPath = new File("data" + File.separator + backUpName);
+        if (!fullPath.exists()) {
+            if (fullPath.mkdir()) {
+                System.out.println("Directory: "
```

Comment:
if I remove `System.out.println()`'s it means I have to remove two if() instructions, and that means only one line stay:
`File fullPath = new File("data" + File.separator + backUpName);`
Is this correct ?